Storage/Storage/Model/WooShippingPackagesResponse+CoreDataProperties.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CoreData
+
+
+extension WooShippingPackagesResponse {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<WooShippingPackagesResponse> {
+        return NSFetchRequest<WooShippingPackagesResponse>(entityName: "WooShippingPackagesResponse")
+    }
+
+    @NSManaged public var siteID: Int64
+    @NSManaged public var allPredefinedOptions: NSOrderedSet?
```

Comment:
I used ordered sets in a couple places since we already did the work to order them in the Networking layer.